regression/test_stress_failovering.py

```diff
@@ -43,14 +43,14 @@ def run_test(self, ka_reqs):
             s.config.set_ka(ka_reqs)
         self.generic_test_routine('cache 0;\n')
 
-    @unittest.expectedFailure
+    #@unittest.expectedFailure
```

Comment:
Dead code. Should be removed. The test is now is in disabled list, good description why the test fails is required.

Reply:
fixed